pkg/authorization/bootstrap/policy.go

```diff
@@ -25,7 +25,7 @@ import (
 // ClusterRoleBindings return default rolebindings to the default roles
 func clusterRoleBindings() []rbacv1.ClusterRoleBinding {
 	return []rbacv1.ClusterRoleBinding{
-		clusterRoleBindingCustomName(rbacv1helpers.NewClusterBinding("edit").Groups("system:kcp:workspace:edit").BindingOrDie(), "system:kcp:workspace:edit"),
+		clusterRoleBindingCustomName(rbacv1helpers.NewClusterBinding("cluster-admin").Groups("system:kcp:workspace:edit").BindingOrDie(), "system:kcp:workspace:edit"),
```

Comment:
Pretty sure this needs to be system:masters or we need to create a new cluster-admin group and add it to the cluster-admin role binding